### PR TITLE
Add Superseed Sepolia and Mainnet

### PR DESCRIPTION
**Pull request by 0xNikolas**
This PR introduces support for the `Superseed Sepolia Testnet` and `Superseed Mainnet `by adding their definitions and exporting them from the relevant files.

**Detailed summary**

- Added a new entry for `superseedSepolia` in `src/chains/index.ts`.
- Added a new entry for `superseed` in `src/chains/index.ts`.
- Created `superseedSepolia` definition in `src/chains/definitions/superseedSepolia.ts` with:
  - ID: `53302`
  - Name: 'Superseed Sepolia'
  - Native currency: Ether (ETH)
  - RPC URL: https://sepolia.superseed.xyz
  - Block explorer: 'Superseed Sepolia Explorer'
  - `Multicall3` contract address
  - Marked as a testnet.
- Created `superseed` definition in `src/chains/definitions/superseed.ts` with:
  - ID: `5330`
  - Name: 'Superseed'
  - Native currency: Ether (ETH)
  - RPC URL: https://mainnet.superseed.xyz
  - Block explorer: 'Superseed Explorer'
  - `Multicall3` contract address

---

## PR-Codex overview
This PR introduces the `Superseed` blockchain configurations for both the mainnet and Sepolia testnet, enhancing the chain definitions in the project.

### Detailed summary
- Added `superseed` export in `src/chains/index.ts`.
- Created `superseed` configuration in `src/chains/definitions/superseed.ts` for mainnet.
- Created `superseedSepolia` configuration in `src/chains/definitions/superseedSepolia.ts` for Sepolia testnet.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/superseed.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      name: 'Superseed Explorer',
+      url: 'https://explorer.superseed.xyz',
+      apiUrl: 'https://explorer.superseed.xyz/api/v2',
+    },
+  },
+  contracts: {
+    ...chainConfig.contracts,
+    disputeGameFactory: {
+      [sourceId]: {
+        address: '0x8b097CF1f9BbD9cbFD0DD561858a1FCbC8857Be0',
```
Comment (jxom) [inline src/chains/definitions/superseed.ts:27]: Can this address please be listed on Superseed's official docs or https://github.com/ethereum-optimism/superchain-registry/tree/main/superchain/configs/mainnet? Needs at least one credible reference.

**Review by jxom — CHANGES_REQUESTED** [review]

**Comment by jxom** [comment]
Please review comment and then reopen the PR.

File: src/chains/definitions/superseed.ts (hunk 1 above)
Comment (0xNikolas) [reply src/chains/definitions/superseed.ts:27]: yes, we'll deploy soon

Comment (0xNikolas) [reply src/chains/definitions/superseed.ts:27]: Added all contracts to docs: https://docs.superseed.xyz/build-on-superseed/contract-addresses/mainnet

**Pull request closed without merging**